docs/prober-and-monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Prober and monitoring
+
+**PMAP is not a official Google product.**
+
+Monitoring consists of two parts: prober and alerting.
+
+## Prober
+
+[Prober](../prober/) is a go binary being deployed to [cloud run
+job](https://cloud.google.com/run/docs/overview/what-is-cloud-run#jobs), and is
+triggered by [cloud scheduler](https://cloud.google.com/scheduler) to constantly
+probing pmap services to check if the services are up.
+
+In each prober job, the prober will upload a yaml file to
```

Comment:
Prober doesn't upload ONE yaml file. You can explain it based on CUJ. One CUJ is policy and the other CUJ is resource mapping

Reply:
updated.